frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -607,37 +574,55 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                         return null
                     }
 
+                    const sessionEventsQuery = hogql`
+                            SELECT uuid, event, timestamp, elements_chain, properties.$window_id, properties.$current_url, properties.$event_type
+                            FROM events
+                            WHERE timestamp > ${start.subtract(BUFFER_MS, 'ms')}
+                              AND timestamp < ${end.add(BUFFER_MS, 'ms')}
+                              AND $session_id = ${props.sessionRecordingId}
```

Comment:
no need to filter by person or distinct id here... the events have the session id or they don't
we don't care what library they come from